etc/kayobe/pulp-repo-versions.yml

```
@@ -1,25 +1,25 @@
 ---
 # Do not edit! This file is autogenerated by Ansible.
 stackhpc_pulp_repo_centos_stream_8_advanced_virtualization_version: 20211122T102435
-stackhpc_pulp_repo_centos_stream_8_appstream_version: 20220817T082321
-stackhpc_pulp_repo_centos_stream_8_baseos_version: 20220817T082321
+stackhpc_pulp_repo_centos_stream_8_appstream_version: 20230201T025809
+stackhpc_pulp_repo_centos_stream_8_baseos_version: 20230201T025809
 stackhpc_pulp_repo_centos_stream_8_extras_version: 20220401T032901
 stackhpc_pulp_repo_centos_stream_8_nfv_extras_version: 20220609T110556
-stackhpc_pulp_repo_centos_stream_8_nfv_openvswitch_version: 20220824T113700
-stackhpc_pulp_repo_centos_stream_8_openstack_xena_version: 20220823T042339
+stackhpc_pulp_repo_centos_stream_8_nfv_openvswitch_version: 20230208T033911
+stackhpc_pulp_repo_centos_stream_8_openstack_xena_version: 20230209T034107
 stackhpc_pulp_repo_centos_stream_8_opstools_version: 20220617T100837
-stackhpc_pulp_repo_centos_stream_8_powertools_version: 20220817T082321
-stackhpc_pulp_repo_centos_stream_8_storage_ceph_pacific_version: 20220803T043053
+stackhpc_pulp_repo_centos_stream_8_powertools_version: 20230201T025809
+stackhpc_pulp_repo_centos_stream_8_storage_ceph_pacific_version: 20230201T025809
 stackhpc_pulp_repo_docker_ce_ubuntu_version: 20220708T132615
-stackhpc_pulp_repo_docker_version: 20220828T050510
-stackhpc_pulp_repo_elasticsearch_logstash_kibana_7_x_version: 20220825T041159
-stackhpc_pulp_repo_epel_modular_version: 20220828T050510
-stackhpc_pulp_repo_epel_version: 20220830T054227
-stackhpc_pulp_repo_grafana_version: 20220831T050502
-stackhpc_pulp_repo_mariadb_10_5_centos8_version: 20220817T082321
-stackhpc_pulp_repo_rabbitmq_erlang_version: 20220828T050510
-stackhpc_pulp_repo_rabbitmq_server_version: 20220803T043053
-stackhpc_pulp_repo_treasuredata_4_version: 20220823T042339
+stackhpc_pulp_repo_docker_version: 20230210T031947
```

```suggestion
stackhpc_pulp_repo_docker_version: 20230203T025251
```
Use same version as from wallaby due to CI failure downloading repo data